main/mainworker.cpp

```diff
@@ -13095,6 +13103,11 @@ bool MainWorker::SetThermostatState(const std::string& idx, const int newState)
 		pGateway->SetProgramState(newState);
 		return true;
 	}
+	else if (pHardware->HwdType == HTYPE_VirtualThermostat)
+	//virtual thermostat set state confor/eco/off/frozen
```

Comment:
Please put the comment inside the { } block

Reply:
ok done